@here/harp-mapview/lib/MapView.ts

```diff
@@ -2659,6 +2691,29 @@ export class MapView extends EventDispatcher {
             : this.m_raycaster.ray.intersectPlane(this.m_plane, cache.vector3[0]);
     }
 
+    /**
+     * Returns the {@link @here/harp-geoutils#GeoCoordinates} from the
+     * given screen position.
+     *
+     * @remarks
+     * This method will always return a point. if the given `(x, y)` value is not intersecting
+     * the ground plane a fallback point on the far plane will be computed.
+     * Therfore the returned point might not be on the earth surface (i.e. altitude > 0)
+     *
+     * @param x - The X position in css/client coordinates (without applied display ratio).
+     * @param y - The Y position in css/client coordinates (without applied display ratio).
+     * @param fallback - Fallback to
```

Comment:
`fallback` not part of the parameter

Reply:
Oh, missed that one. Initially I was thinking about adding an additional parameter instead of creating a new function but I did not manage to convince TypeScript to have a different return type depending on the value of that parameter. Let me try a bit more.